packages/kbn-ai-playground/server/routes.ts

```diff
@@ -0,0 +1,150 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { schema } from '@kbn/config-schema';
+import { fetchFields } from '../lib/fetch_query_source_fields';
+import Assist, { ConversationalChain, Prompt } from '@elastic/ai-assist';
```

Comment:
I thought the idea is to take out server codes from `elastic/ai-assist` and move it into our own server codes so we do not have any more dependency on `elastic/ai-assist`. Please correct me if I am wrong.

Reply:
Yeah, I got it wrong)